cgi-bin/Apache/LiveJournal.pm

```diff
@@ -95,7 +95,7 @@ sub handler
             # by the fact that mod_proxy did nothing, requiring mod_proxy_add_forward, then
```

Comment:
Yeah, I thought about removing it, and think you may be right. I didn't feel like I had the :brain: to sit down and puzzle it all out when I was working on it though...